.github/workflows/quality.yml

```diff
@@ -203,3 +203,26 @@ jobs:
           name: detekt-report
           path: |
             */build/reports/detekt/detekt.html
+
+  towncrier:
+    name: Towncrier check
+    runs-on: ubuntu-latest
+    if: always() && github.event_name == 'pull_request'
```

Comment:
Other thought is whether you want this to apply if the target is not `develop` - eg PRs against a release branch?

Reply:
the always() in other examples is if the step depends on other steps; it might fail to run because those others failed; but this is entirely parallel.

(though, it might be good to have this check at the start of a tree of checks - only bother running the slow tests once the fast tests are passing, as otherwise you will need to re-run the slow checks again when the changelog file has been committed)

Reply:
OK, thanks for explanation. Adding `always()` was an attempt (9335886a5af70f07deb61799dffd5120a4e52079) to trigger this action for this particular PR. I will drop this commit.

> if the target is not develop

In this case I think we do not want to run this check. But I have no idea how to do that. Will do some research.